apps/site/src/pages/account/api-keys-panel/api-keys-list.tsx

```diff
@@ -0,0 +1,201 @@
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from "@mui/material";
+
+import { apiClient } from "../../../lib/api-client";
+import { useApiKeys } from "./api-keys-context";
+import { ApiKeyCard } from "./api-keys-list/api-key-card";
+import { ApiKeyTableRow } from "./api-keys-list/api-key-table-row";
+import { MobileApiKeyItem } from "./api-keys-list/mobile-api-key-item";
+import { RevokeApiKeyCard } from "./api-keys-list/revoke-api-key-card";
+import { ApiKeyItemProps, ApiKeyProps } from "./types";
+
+export const ApiKeysList = () => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
```

Comment:
It's worth saying if we ever SSR this page, this will cause layout shift post-hydration, and we may want to switch which we're rendering by CSS instead of using this, for future stability. See https://github.com/blockprotocol/blockprotocol/pull/1097

Reply:
@nonparibus this page is not SSR rendered for now, not sure if it's worth the effort of moving to media queries here. I agree with doing do, but maybe a follow-up ticket for this would be better?

(IIRC there are few other places I used it like this, noting so we don't forget others if we decide to follow-up)

Reply:
I think useMediaQuery is generally  a bad pattern, and it makes it difficult to introduce SSR if you want to. It wouldn't be a big or difficult change to render both and hide/show based on CSS, and not introduce a potential blocker. There should be a strong justification for using useMediaQuery that I just don't see here.

Reply:
Replaced with `sx` prop here: https://github.com/blockprotocol/blockprotocol/pull/1257/commits/2591917ec508d48322cdf3e9d5817a5cba3bc224